frontend/src/app/testResults/TestResultsList.test.tsx

```diff
@@ -279,6 +279,28 @@ describe("TestResultsList", () => {
         "Cragell, Barb Whitaker"
       );
     });
+
+    it("should be able to filter by condition value", async () => {
+      const { user } = renderWithUser();
+      expect(
+        await screen.findByText("Test Results", { exact: false })
+      ).toBeInTheDocument();
+      expect(
+        await screen.findByText("Cragell, Barb Whitaker")
+      ).toBeInTheDocument();
+      expect(await screen.findByText("Gerard, Sam G"));
+      expect(
+        await screen.findByRole("option", { name: "Negative" })
```

Comment:
remove `expect(await screen.findByRole("option", { name: "Negative" })`
since it doesnt add value to the test

Reply:
Good catch 👁️ https://github.com/CDCgov/prime-simplereport/pull/6682/commits/cc7f1bf0b930c9d484e035cd2a54f991141b303f